### PR TITLE
#2970 MachineInfiniteBus in dynawo, examples with appropriate icon an…

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
closes [#2970](https://github.com/dynawo/dynawo/issues/2970)

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/MachineInfiniteBus.mo [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  connect(gridImpedance.terminal1, infiniteBus.terminal) annotation(
+    Line(points = {{-52, 0}, {-62, 0}, {-62, -4.26324e-14}, {-92, -4.26324e-14}}, color = {0, 0, 255}));
+  connect(load.terminal, gridImpedance.terminal2) annotation(
+    Line(points = {{0, -40}, {0, 0}, {-12, 0}}, color = {0, 0, 255}));
+  connect(load.PRefPu, PRefPu.y) annotation(
+    Line(points = {{-18, -70}, {-6, -70}, {-6, -48}}, color = {0, 0, 127}));
+  connect(load.QRefPu, QRefPu.y) annotation(
+    Line(points = {{20, -70}, {6, -70}, {6, -48}}, color = {0, 0, 127}));
+
+  annotation(preferredView = "diagram",
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/MachineInfiniteBus.mo:159]: If we put it in dynawo, then we should describe the test case

**Review by cossartque — CHANGES_REQUESTED** [review]

**Review by joyelfeghali — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/MachineInfiniteBus.mo [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model MachineInfiniteBus "Synchronous machine infinite bus with constant regulations"
```
[the remark this answers is not in the document]
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/MachineInfiniteBus.mo]: Done

**Review by joyelfeghali — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIBStepEfdPm.mo [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+    Line(points = {{-52, 0}, {-62, 0}, {-62, -4.26324e-14}, {-92, -4.26324e-14}}, color = {0, 0, 255}));
+  connect(load.terminal, gridImpedance.terminal2) annotation(
+    Line(points = {{0, -40}, {0, 0}, {-12, 0}}, color = {0, 0, 255}));
+  connect(load.PRefPu, PRefPu.y) annotation(
+    Line(points = {{-18, -70}, {-6, -70}, {-6, -48}}, color = {0, 0, 127}));
+  connect(load.QRefPu, QRefPu.y) annotation(
+    Line(points = {{20, -70}, {6, -70}, {6, -48}}, color = {0, 0, 127}));
+
+  annotation(preferredView = "diagram",
+    experiment(StartTime = 0, StopTime = 30, Tolerance = 0.000001),
```
[the remark this answers is not in the document]
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIBStepEfdPm.mo:160]: Agreed for stopTime = 50 s. The active and reactive load is already mentioned in the documentation, do you want a comment on load.UPu ?

[the remark this answers is not in the document]
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIBStepEfdPm.mo]: I have added a comment to the documentation, saying that P and Q of the load are proportional to the voltage.

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIBStepEfdPm.mo [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model SMIBStepEfdPm "Synchronous machine infinite bus with steps on Efd and Pm"
```
Comment (joyelfeghali) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIBStepEfdPm.mo:16]: Should we change SMIB.Resources.Images.CMakeLists.txt to add the new figures?

**Pull request merged**